### PR TITLE
Add Docker compile Support for Pipelines

### PR DESCRIPTION
**Pull request by w5688414**
### PR types

+  New features

### PR changes

+ Docs

### Description

+ Add Docker compile Support for Pipelines

### CODE REVIEW

File: pipelines/docker/README.md [hunk 1]
```diff
@@ -59,3 +59,66 @@ docker-compose -f docker-compose-gpu.yml stop
...
+nvidia-docker run --name pipelines --net host --shm-size 4g -it registry.baidubce.com/paddlepaddle/paddle:2.3.2-gpu-cuda11.2-cudnn8 /bin/bash
+git clone https://github.com/PaddlePaddle/PaddleNLP.git
+cd PaddleNLP/pipelines/
+pip install -r requirements.txt -i https://pypi.tuna.tsinghua.edu.cn/simple
+python setup.py install
+apt-get install lsof
+```
+镜像构建完成可以使用`Ctrl+P+Q`组合键跳出容器。
+
+在第一步构建镜像的过程中，如果是Cuda的其他版本，则需要在[官网](https://www.paddlepaddle.org.cn/install/quick?docurl=/documentation/docs/zh/install/docker/linux-docker.html)上查找是否有对应的CUDA版本的Docker，如果没有，则需要自己手工构建一个该CUDA版本的Docker，然后安装对应CUDA版本的PaddlePaddle，然后继续执行上面的流程。
```
Comment (tianxin1860) [inline pipelines/docker/README.md]: Cuda 写法统一 CUDA

**Review by tianxin1860 — DISMISSED** [review]
Leave a comment

File: pipelines/docker/README.md (hunk 1 above)
Comment (w5688414) [reply pipelines/docker/README.md]: 已修改

**Review by tianxin1860 — APPROVED** [review]
LGTM

**Pull request merged**